src/dynamic-side-content.scss

```diff
@@ -0,0 +1,83 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-dynamic-side;
+
+.#{$block} {
+  $main: #{$block}__main;
+  $side: #{$block}__side;
+
+  $main-width-screen-sm: 100%;
+  $main-width-screen-md: 66.666%;
+  $main-width-screen-xl: 75%;
+
+  @include fd-reset();
+  @include fd-flex();
+
+  width: 100%;
+
+  .#{$main} {
+    @include fd-reset();
+
+    width: $main-width-screen-xl;
+  }
+
+  .#{$side} {
+    @include fd-reset();
+
+    flex: 1 1 0;
+  }
+
+  &--sm {
+    .#{$main} {
+      width: $main-width-screen-sm;
+    }
+
```

Comment:
You can create a mixin
```
@mixin fd-dynamic-side-main($width: 100%) {
    .#{$block}__main {
         width: $width;
    }
}
```

and then inside the modifier classes you just include the mixins:
```
&--sm {
    @include fd-dynamic-side-main()
}

&--md {
    @include fd-dynamic-side-main(66,666%)
}

```

The advantage of this approach is that you don't have to create the variables and then you don't repeat 
```
.#{$main} {
      width: some value;
    }
```

But this is just a personal preference. I wanted to give you another point of view. Your code looks ok and you don't have to change anything :)

Reply:
Thanks, got it!
I switched to mixin approach to encapsulate class name